README.md

```diff
@@ -1,97 +1,41 @@
-before running the script.
-
-To deploy the script:
-
-1.  Copy it into the
-    [Google Ads script editor](https://support.google.com/google-ads/answer/188712)
-
-2.  Remove the `exports` block at the bottom of the script.
-
-3.  Edit the script's constants and parameters to reflect your preferences and
-    requirements.
-
-4.  Test that the script will preform the correct actions by using the preview
-    function of the scripts editor.
-
-5.  *Optional*
-    [Schedule the script](https://support.google.com/google-ads/answer/188712?hl=en#:~:text=run%20it%20again.-,Scheduling%20a%20script,-Once%20you%27ve%20created)
-    to run on a regular basis automatically.
-
-Running the script will result in keywords being added to the account, if any appropriate search terms are found. If email addresses are added to the configuration, an email will also be sent to the recipients with the results of the execution.
+Configuration
+The following constants can be changed to affect how the script executes. They are also documented in the script itself.
 
-## Configuration
+**MANDATORY + BASIC Parameters**
 
-The following constants can be changed to affect how the script executes. They are also documented in the script itself.
+SCRIPT_NAME : The name used to identify the script in logs and the results email.
 
-SCRIPT_NAME
-: The name used to identify the script in logs and the results email.
+CAMPAIGNS : An array of Campaign names to extract search terms from. All Ad Groups within these campaigns will be processed. To include all campaigns, set value to ['__ALL__'] . If empty, the ADGROUPS constant will be used instead.
 
-CAMPAIGNS
-: An array of campaign names to extract search terms for. All of the child Ad Groups of the listed campaigns will be included. If all campaigns should be included, you can use `__ALL__` as the only element of the array. If this constant is empty, the `ADGROUPS` constant will be used instead when building the search term report.
+ADGROUPS : An array of Ad Group names to extract search terms from. Only used if the CAMPAIGNS constant is an empty array.
 
-ADGROUPS
-: Only used if the CAMPAIGNS constant is an empty array. An array of Ad Groups to extract search terms for and to add the resulting keywords too. 
+ENABLE_KEYWORDS : If true, new keywords will be enabled immediately. Otherwise, the keywords are added in a PAUSED state, which allows for a review prior to enabling.
+This is not applicable to negative keywords (which are always enabled).
 
-ADD_KEYWORDS_TO_DIFFERENT_ADGROUP
-: A boolean flag to specify that keywords should be added to a specific Ad Group, specified by the `ADGROUP_TO_ADD_SEARCH_KEYWORDS` constant.
+GAQL_QUERY_SEARCH_TERM : Ths is where you define the criteria for "High Performance." The GAQL query will be used to extract the applicable search terms. Levarage GAQL Query Builder to help you create a query with a different set of filters, if needed.
```

Comment:
spelling mistake in "Ths". Sublime edit has a built-in spell checker you could use here.